Source/OxyPlot/Graphics/Element.Selectable.cs

```diff
@@ -13,24 +13,15 @@ namespace OxyPlot
     using System.Collections.Generic;
 
     /// <summary>
-    /// Provides an abstract base class for elements that support selection.
+    /// Provides functionality to support selection.
```

Comment:
Does having different documentation for each declaration of a partial class create confusion? There is some StyleCop suppression on `PlotModel` in PlotModel.Rendering.cs; `PlotModel` is only documented in PlotModel.cs